src/Client.php

```diff
@@ -79,7 +79,7 @@ public function addChoicesMapping(array $mapping = []): void
     {
         foreach ($mapping as $fieldStringId => $choices) {
             if (is_array($choices)) {
-                if ( ! array_key_exists($fieldStringId, $this->choicesMapping)) {
+                if (! array_key_exists($fieldStringId, $this->choicesMapping)) {
```

Comment:
We've had back and forth in this rule... seems it's going back again now.
Would be nice to keep aligned with the MM repo